cmd/node/config/config.toml

```diff
@@ -926,3 +926,9 @@
         Capacity = 1000
         Type = "SizeLRU"
         SizeInBytes = 314572800 #300MB
+
+# Changing this config is not backwards compatible
+[SoftAuctionConfig]
+    TopUpStep = "10000000000000000000" # 10 EGLD
+    MinTopUp = "1" # 0.00...01 EGLD , should be very low, but != zero
```

Comment:
Yes, if we decide for example for later to only select nodes which have a topUp>minTopUp => selection will be done for less validators => improved time performance.
Even if all validators have 2500 EGLD, therefore 0 topUp, algorithm would still work. We would fail to fill auction slots => we will simply order all validators by blsKey xor rand and fill the avaiable auction slots. This behavior is also demonstrated in `TestAuctionListSelector_calcSoftAuctionNodesConfigEdgeCases` test (see _"two validators, both have zero top up"_ )

Reply:
ok